docs/semgrep-ci/configuring-blocking-and-errors-in-ci.md

```diff
@@ -16,7 +16,7 @@ This article documents how Semgrep pipelines handle blocking findings and errors
 
 ## Default configuration of blocking findings and error suppression
 
-Semgrep blocks the pull requests (PRs) or merge requests (MRs) in its default configuration only when it matches a blocking finding.
+Semgrep can help block pull requests (PRs) or merge requests (MRs). Semgrep returns exit code `1`, and you can use this result to set up additional checks to enforce a block on a PR or MR when it matches a blocking finding. This action applies to both full scans and [diff-aware scans](/semgrep-code/glossary#diff-aware-scan).
```

Comment:
```suggestion
Semgrep can help block pull requests (PRs) or merge requests (MRs). Semgrep returns exit code `1`, and you can use this result to set up additional checks to enforce a block on a PR or MR when it matches a blocking finding. This action applies to both full scans and [diff-aware scans](/semgrep-code/glossary#diff-aware-scan).
```

Just curious - where can the user set up additional checks? Is this something they do on the CI side (i.e. dependent on the CI provider?)

Reply:
Yeah, it's dependent on the provider -- Semgrep throws a 1, and the user can decide how they want their CI system to handle this (or not)